shell/platform/tizen/channels/text_input_channel.cc

```diff
@@ -318,34 +161,42 @@ void TextInputChannel::HandleMethodCall(
       result->Error(kBadArgumentError, "Could not set client, ID is null.");
       return;
     }
+
     if (client_config.IsNull()) {
       result->Error(kBadArgumentError,
                     "Could not set client, missing arguments.");
     }
+
     client_id_ = client_id_json.GetInt();
     input_action_ = "";
     auto input_action_json = client_config.FindMember(kTextInputAction);
+
     if (input_action_json != client_config.MemberEnd() &&
         input_action_json->value.IsString()) {
       input_action_ = input_action_json->value.GetString();
     }
+
     input_type_ = "";
     auto input_type_info_json = client_config.FindMember(kTextInputType);
+
```

Comment:
I'm perfectly fine with these changes but just want to note that Google guides you to minimize the use of vertical whitespaces (just in case you didn't notice).

https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace

Reply:
Thank you for notice, but the implementation of this function was too inconvenient to read for me
In addition to adding vertical whitespaces, I will avoid unnecessary changes as much as possible.